tests/components/dsmr/test_config_flow.py

```diff
@@ -49,13 +49,69 @@ async def test_setup_network(hass, dsmr_connection_send_validate_fixture):
     with patch("homeassistant.components.dsmr.async_setup_entry", return_value=True):
         result = await hass.config_entries.flow.async_configure(
             result["flow_id"],
-            {"host": "10.10.0.1", "port": 1234, "dsmr_version": "2.2"},
+            {
+                "host": "10.10.0.1",
+                "port": 1234,
+                "dsmr_version": "2.2",
+            },
         )
```

Comment:
Please block till done after the last configure call for all tests that finish with create entry results.

Reply:
Update all test cases.